libr/anal/p/anal_arm_cs.c

```diff
@@ -2584,21 +2587,36 @@ static void anop64(csh handle, RAnalOp *op, cs_insn *insn) {
 	}
 }
 
-static ut64 lookahead(csh handle, const ut64 addr, const ut8 *buf, int len, int distance) {
-	cs_insn *insn = NULL;
-	int n = cs_disasm (handle, buf, len, addr, distance, &insn);
-	if (n < 1) {
-		return UT64_MAX;
+static void anal_itblock(RAnal *a, cs_insn *insn) {
+	if (!ht_it) {
+		ht_it = ht_uu_new0 ();
+	}
+	size_t i;
+	for (i = 1; i < 5; i++) {
+		switch (insn->mnemonic[i]) {
+		case 0x74: //'t'
+			if (!ht_uu_insert (ht_it, insn->address + (i * insn->size), insn->detail->arm.cc)) {
```

Comment:
just use `ht_uu_update` if you want to insert or update.
From the ht_inc.h file:
```
// Insert a new Key-Value pair into the hashtable, or updates the value if the key already exists.
SDB_API bool Ht_(update)(HtName_(Ht)* ht, const KEY_TYPE key, VALUE_TYPE value);
```

Reply:
Done